.github/workflows/promptflow-parallel-e2e-test.yml

```diff
@@ -0,0 +1,87 @@
+name: promptflow-parallel-e2e-test
+
+on:
+  schedule:
+    - cron: "40 10 * * *" # 2:40 PST every day
+  pull_request:
+    paths:
+      - src/promptflow/**
+      - src/promptflow-core/**
+      - src/promptflow-tracing/**
+      - src/promptflow-parallel/**
+      - .github/workflows/promptflow-parallel-e2e-test.yml
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: read
+
+env:
+  IS_IN_CI_PIPELINE: "true"
+  WORKING_DIRECTORY: ${{ github.workspace }}/src/promptflow-parallel
+
+jobs:
+  test:
```

Comment:
create a special name for each job name.
This is important handler for github.

Reply:
also the other one.

Reply:
renamed